packages/toast-ui.grid/docs/en/data-source.md

```diff
@@ -118,6 +118,7 @@ The response data must be in JSON string format. When the request has been succe
 - **result** `{boolean}`: It will return `true` when the request has been successfully made, and `false` otherwise. 
 - **data**
     - **content** `{array}`: As an array of row data, it is the same as the `grid.resetData()` method's parameter. 
+    - **pagination** `{object}`: with two properties
```

Comment:
There is an explanation below, so I don't think there's any explanation in the **pagination**.

Reply:
sorry, I don't remember the context of this PR and I don't understand your comment. Can you clarify?

Reply:
@joebordes 
Oh, no, you can just ignore it.I'll merge this! Thanks :)